### DISCUSSION TITLE
Manufacturing Order status icons

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#626] [Q&A]
Hi,

after new release 8.30 i've noticed that modifying Manufacturing Order status through its icons is not possible anymore.

![image](https://github.com/frePPLe/frepple/assets/129165989/c9656432-08c3-400a-9e7d-69c1df4d6bb2)

Is it a bug?

**Comment by jdetaeye** [comment]
Yes, it is a bug
